### ISSUE TITLE
Feature request: async/await support

### ISSUE THREAD

**Issue by peltsi212** [angelozerr/tern.java#453]
Node 8 has been released 2017-05-30 and it supports async/await
eslint support ES2017
http://eslint.org/blog/2016/09/eslint-v3.6.0-released
Is it possible to upgrade eslint and get support for async/await to tern?

**Comment by FireController1847** [comment]
PLEASE I NEED THIS, there's so many bugs because this isn't supported and I don't want to keep using Atom to code, I want to use Eclipse but I can't until this gets added!

**Comment by angelozerr** [comment]
> get support for async/await to tern?

It's an issue for ternjs https://github.com/ternjs/tern/issues/814

**Comment by peltsi212** [comment]
It seems that there is no progress in ternjs 
eslint is already integrated to VSCode editor and works fine and supports ES2017 async/await
It would be nice to have eslint integrated also to eclipse.

**Comment by FireController1847** [comment]
Yeah, I just gave up on eclipse and moved to VSCode. Works much better 👌
